src/dyn_model.jl

```diff
@@ -220,22 +246,31 @@ Notes:
 - The variables that are not in the diagonal block for the Jacobian are νs_next and ν
 """
 function kkt_storage(
-    s, s_prev, ch, dis, λsu, λsl, λchu, λchl, λdisu, λdisl, ν, νs_t, νs_next, P, C, η_c, η_d
-    )
+    s, s_prev, ch, dis, λsu, λsl, λchu, λchl, λdisu, λdisl, λrampl, λrampu, ν, νs_t, νs_next, P, C, η_c, η_d,
+    ρ, gt, gt_prev
+)
     return [
         (λsu - λsl) + (νs_next - νs_t);  # ∇_s L
-        (λchu - λchl) + ν * η_c + νs_t * η_c ;  # ∇_ch L
-        (λdisu - λdisl) - ν/η_d - νs_t/η_d;  # ∇_dis L
+        (λchu - λchl) + ν + νs_t * η_c ;  # ∇_ch L
+        (λdisu - λdisl) - ν - νs_t/η_d;  # ∇_dis L
         λsl .* (-s);
         λsu .* (s - C);
         λchl .* (-ch);
         λchu .* (ch-P);
         λdisl .* (-dis);
         λdisu .* (dis-P);
+        λrampl .* (gt_prev - ρ - gt);
+        λrampu .* (gt - gt_prev - ρ);
```

Comment:
Might not be very important -- but it find it weird to put this in `kkt_storage`. It does not make much sense to me? 

Or we should recall it `kkt_dynamic` to emphasize that it is all the KKT terms that have something to do with time dependency?

Reply:
Or what not include these in `kkt_ramp`?

Reply:
I'm hesitant to call it `kkt_dynamic`, since the main function is already `kkt_dyn`. And putting them in `kkt_ramp` would definitely increase the code complexity a bit.

I'm going to leave it in `kkt_storage` for now, but I agree we should find a better name for it.